src/routes/v1/article.route.js

```diff
@@ -0,0 +1,30 @@
+import { Router } from 'express';
+import ArticleController from '../../controllers/articles.controller';
+import authMiddleware from '../../middleware/auth.middleware';
+import validationMiddleware from '../../middleware/validation.middleware';
+
+const router = Router();
+const validateRequest = validationMiddleware();
+
+router.post(
+  '/articles',
+  authMiddleware,
+  validateRequest,
+  ArticleController.createArticle
+);
+router.get('/articles', ArticleController.fetchAllArticles);
```

Comment:
Good job @tobechukwuobitube , Please use the pagination feature that was implemented by @Onyimatics to help the users limit the fetch requests

Reply:
Thank you @tolumide-ng for this review.
However, the pagination feature you are suggesting is not within the scope of this [task](https://www.pivotaltracker.com/story/show/166840922).